cfme/tests/automate/custom_button/test_cloud_objects.py

```diff
@@ -369,29 +369,21 @@ def test_custom_button_expression_cloud_obj(
 
     for setup_obj in setup_objs:
         view = navigate_to(setup_obj, "Details")
-        custom_button_group = Dropdown(view, group.hover)
-
-        # Note: For higher version (5.10+), button group having single button;
-        # If button is disabled then group disabled.
+        custom_button_group = Dropdown(view, group.text)
 
+        # TODO(ndhandre): add hover check if group disabled
```

Comment:
Why not include that in this PR?

Reply:
@mshriver I thought it will take time as `wt.pfy` also not that much required as I need to enhance test and function taking care about `disabled` or `enabled` text input.

Reply:
wt.pf 0.1.2 released with dropdown `hover` attribute.  Not clear from your statement above - do you want to try using that wt.pf update now, or merge this with TODO's?